e2e/testcafe-devextreme/tests/editors/list/grouping.ts

```diff
@@ -53,6 +54,37 @@ test('Grouped list appearance', async (t) => {
   },
 }));
 
+test('Grouped List last item of last group should have margin-bottom = 4px', async(t) => {
```

Comment:
margin-bottom = 4px it's actual value for Fluent theme only. I think that name can be more common with description what appearance testing.